app/assets/stylesheets/modules/subnavbar.scss

```diff
@@ -25,7 +25,15 @@
     color: $white;
   }
 
+  .sul-services-menu {
+    min-width: 300px;
+  }
+
   // For all subnavbar navbar lists
+  ul ul ul {
```

Comment:
Should/could any of these tags be qualified (with a class)?

Reply:
@jcoyne thanks for this one, I've updated it to `.sul-services-menu ul` to better capture the idea that it's a sublist for a specific menu.